account_financial_report_qweb/__openerp__.py

```diff
@@ -35,8 +35,11 @@
         'report/templates/trial_balance.xml',
         'view/account_view.xml'
     ],
-    'test': [
-    ],
+    'external_dependencies': {
+        "python": [
+            "xlsxwriter",
+        ],
+    },
```

Comment:
@Yajo indeed I had not seen that part https://github.com/OCA/account-financial-reporting/blob/58c559b59fd0d28dc4ee9c824365c2c579b491ff/account_financial_report_qweb/report/abstract_report_xlsx.py#L38

IMO it is not the best way to do it because this override of `create_xlsx_report` does not call super(). @jcoux I think reporting-engine must be extended to support options.